pkg/projects/functions.go

```diff
@@ -74,3 +77,40 @@ func SendCommandSSH(ip *string, port *int, user, password, command string, foreg
 		return nil
 	}
 }
+
+//SendCommandSCP try to access SSH with timer and sends command
+func SendCommandSCP(ip *string, port *int, user, password, filename, destpath string) ProcTimerFunc {
+	return func() error {
+		if *ip != "" {
+			configSSH := &ssh.ClientConfig{
+				User: user,
+				Auth: []ssh.AuthMethod{
+					ssh.Password(password),
+				},
+				HostKeyCallback: ssh.InsecureIgnoreHostKey(),
+				Timeout:         defaults.DefaultRepeatTimeout,  //no timeout
+			}
+			conn, err := ssh.Dial("tcp", fmt.Sprintf("%s:%d", *ip, *port), configSSH)
```

Comment:
I guess it's fine for now, while we have very low commands traffic via ssh, and the remote side is actually `localhost`. But it in the future it might be a performance bottleneck. So might be worth to cache the connection (not in this commit though, and maybe not even in this PR)